pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -398,8 +398,22 @@ def default_compilation_pass(self, optimisation_level: int = 2) -> BasePass:
         # then please update this page accordingly
         # https://cqcl.github.io/pytket-quantinuum/api/index.html#default-compilation
         # Edit this docs source file -> pytket-quantinuum/docs/intro.txt
+
         if optimisation_level == 0:
-            passlist.append(self.rebase_pass())
+
+            def replace_swaps(c: Circuit) -> Circuit:
+                c.replace_SWAPs()
+                return c
+
+            passlist.extend(
+                [
+                    CustomPass(lambda c: replace_swaps(c)),
+                    auto_rebase_pass({OpType.Rz, OpType.PhasedX, OpType.TK2}),
```

Comment:
This will break native gates, for example
```python
b = QuantinuumBackend("", machine_debug=True)
c = Circuit(2).ZZMax(0, 1)
d = b.get_compiled_circuit(c, optimisation_level=0)
d.n_gates #26
```